bulk-user-setup/bulk-user-setup.config

```diff
@@ -47,13 +55,25 @@ NORMAL=$(tput sgr0)
 ###########################################
 
 #'|' delimited string of roles. These roles will be added by default for all caseworkers
-DEFAULT_ROLES="caseworker"
+#DEFAULT_ROLES="caseworker"
+#DEFAULT_CASEWORKER_ROLE="caseworker"
+
+IGNORED_ROLES_FROM_USER_ADD_REQUEST="caseworker"
+
+IGNORED_ROLES_FROM_USER_DELETE_REQUEST="caseworker|payments|payments-refund|payments-refund-approver"
+
+#'|' delimited string of roles. These roles will be added by default for all caseworkers
+ADD_ROLES_BY_DEFAULT="caseworker"
 
-DEFAULT_CASEWORKER_ROLE="caseworker"
+#'|' delimited string of roles. These roles will be deleted by default for all caseworkers
```

Comment:
We can change the comment to 
**These roles will be deleted by default for all caseworkers when there is no other caseworker related roles available for that user**

Reply:
Done